src/main/java/com/synopsys/integration/jenkins/scan/service/BridgeDownloadParametersService.java

```diff
@@ -28,25 +30,25 @@ public boolean performBridgeDownloadParameterValidation(BridgeDownloadParameters
         boolean validInstallationPath = isValidInstallationPath(bridgeDownloadParameters.getBridgeInstallationPath());
 
         if(validUrl && validVersion && validInstallationPath) {
-            listener.getLogger().println("Bridge download parameters are validated successfully.");
+            listener.getLogger().println(LogMessages.BRIDGE_DOWNLOAD_PARAMETERS_VALIDATED_SUCCESSFULLY);
             return true;
         } else {
-            listener.getLogger().println("Bridge download parameters are not valid.");
+            listener.getLogger().println(LogMessages.INVALID_BRIDGE_DOWNLOAD_PARAMETERS);
             return false;
         }
     }
 
     public boolean isValidUrl(String url) {
         if (url.isEmpty()) {
-            listener.getLogger().println("The provided Bridge download URL is empty.");
+            listener.getLogger().println(LogMessages.EMPTY_BRIDGE_DOWNLOAD_URL_PROVIDED);
```

Comment:
Sometimes we are using printf and sometimes we are using println. Shouldn't we follow the same convention?

Reply:
When we don't need args to pass for formatting, we can simply use `println()`.
You can see we used `printf()` in case of passing args